src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/SourcePacketExtension.java

```diff
@@ -121,12 +127,87 @@ public long getSSRC()
     public void setSSRC(long ssrc)
     {
         if (ssrc == -1)
+        {
             removeAttribute(SSRC_ATTR_NAME);
+        }
         else
+        {
             setAttribute(SSRC_ATTR_NAME, Long.toString(0xffffffffL & ssrc));
+        }
+    }
+
+    /**
+     * Check if this source has an ssrc
+     *
+     * @return true if it has an ssrc, false otherwise
+     */
+    public boolean hasSSRC()
+    {
+        return getAttributeAsString(SSRC_ATTR_NAME) != null;
+    }
+
+    /**
+     * Gets the rid of this source, if it has one
+     *
+     * @return the rid of the source or null
+     */
+    public String getRid()
```

Comment:
i'm in agreement with @ibauersachs here for the reasons he pointed out, in my experience this capitalization leads to fewer awkward names.